HTTPServerProject.Tests/HTTPServerProxyTests/ResponseTests.cs

```diff
@@ -0,0 +1,178 @@
+using HTTPServerReadTests.Streams;
+using HTTPServerProxy.Response;
+using HTTPServerWriteTests.Streams;
+
+namespace HTTPServerProxyTests.Response;
+
+public class UnitTestsForProxyResponse
+{
+    [Fact]
+    public void GetTodoPOSTValidResponseTest()
+    {
+        var expectedStatusLine = "HTTP/1.1 201 Created";
+        var proxyStream = new List<string>() { "HTTP/1.1 200 OK", "Content-Type: application/json", "Content-Length: 20", "", "{ 'task': 'First Item' }" };
```

Comment:
These lists should broken up into multiple lines - they are really long.

Reply:
Agreed. Will update